diff --git a/node_modules/@rbxts/testez/src/Reporters/TextReporter.lua b/node_modules/@rbxts/testez/src/Reporters/TextReporter.lua
index e40d858..2d274c3 100644
--- a/node_modules/@rbxts/testez/src/Reporters/TextReporter.lua
+++ b/node_modules/@rbxts/testez/src/Reporters/TextReporter.lua
@@ -95,7 +95,11 @@ function TextReporter.report(results)
 		print("")
 
 		for _, message in ipairs(results.errors) do
-			TestService:Error(message)
+			if not _G.LUNE_TEST then
+				TestService:Error(message)
+			else
+				stdio.ewrite(`{message}\n`)
+			end
 
 			-- Insert a blank line after each error
 			print("")
